core/templates/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -465,6 +471,11 @@ angular.module('oppia').directive('questionsList', [
             ctrl.editorIsOpen = true;
             var groupedSkillSummaries = ctrl.getGroupedSkillSummaries();
             var selectedSkillId = ctrl.selectedSkillId;
+            if (newQuestionIsBeingCreated) {
+              ContextService.setImageSaveDestinationToLocalStorage();
+            } else {
+              ImageLocalStorageService.flushStoredImagesData();
```

Comment:
Why is this in else? Shouldn't we flush it before a new question is created always?

Reply:
done, changed